src/commands/DebuggerCommands.ts

```diff
@@ -29,28 +29,20 @@ export namespace DebuggerCommands {
 
     // const workspaceFolder = workspace.getWorkspaceFolder(workspaceUri);
 
-    if (debugNetwork.isLocalNetwork()) {
```

Comment:
Is this method used anywhere else? If not, I guess would be safe to delete it.

Reply:
Ok, gonna check it

Reply:
Removed! Thanks! :)